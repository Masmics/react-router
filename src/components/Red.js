import React from 'react';

function Red() {
  return (
    <>
      <h1>Red!</h1>
      <div style={{ width: '100px', height: '100px', backgroundColor: '#ff0000' }}></div>
    </>
  );
}

export default Red;

