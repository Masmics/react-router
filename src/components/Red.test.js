import React from 'react';
import { shallow } from 'enzyme';
import Red from './Red';

describe('Red component', () => {
  it('renders the red', () => {
    const wrapper = shallow(<Red />);
    expect(wrapper).toMatchSnapshot();
  });
});
