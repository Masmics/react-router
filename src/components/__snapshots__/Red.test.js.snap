// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Red component renders the red 1`] = `
<Fragment>
  <h1>
    Red!
  </h1>
  <div
    style={
      Object {
        "backgroundColor": "#ff0000",
        "height": "100px",
        "width": "100px",
      }
    }
  />
</Fragment>
`;
